config/initializers/postgres_required_versions.rb

```diff
@@ -5,14 +5,13 @@ def initialize(*args)
   end
 
   def check_version
-    msg = "The version of PostgreSQL being connected to is incompatible with #{Vmdb::Appliance.PRODUCT_NAME} (10 required)"
+    msg = "The version of PostgreSQL being connected to (#{postgresql_version}) is incompatible with #{Vmdb::Appliance.PRODUCT_NAME} (130000 / 13 required)"
 
-    if postgresql_version < 90500
+    if postgresql_version < 10_00_00
       raise msg
     end
 
-    if postgresql_version < 100000 || postgresql_version >= 130000
-      raise msg if Rails.env.production? && !ENV["UNSAFE_PG_VERSION"]
+    if postgresql_version >= 14_00_00
```

Comment:
We no longer raise if you try to run PG 14+ in production, we just log a warning.